pages/evening.js

```diff
@@ -0,0 +1,47 @@
+import Link from "next/link";
+import QuestionForm from "../components/QuestionForm";
+import styled from "styled-components";
+
+export default function EveningForm({ eveningAnswers, onAddEveningAnswer }) {
+  return (
+    <>
+      <StyledLink href="/">Home</StyledLink>
+      <Heading>Guten Abend.</Heading>
+      <StyledQuestion>Was war heute das Beste des Tages?</StyledQuestion>
```

Comment:
This should be part of your `QuestionForm` in form of a `label` - as you had it before. You can change the content of the label by using a prop :)